integrations/aerospike/prometheus_metadata.yaml

```diff
@@ -0,0 +1,40 @@
+platforms:
+  - type: GKE
+    launch_stage: GA
+    detections:
+      - characteristic_metric:
+          metric_type: prometheus.googleapis.com/aerospike_namespace_memory_used_bytes/gauge
+    exporter_metadata:
+      name: Aerospike Prometheus Exporter
+      doc_url: https://github.com/aerospike/aerospike-prometheus-exporter
+      minimum_supported_version: "1.8.0"
+    default_metrics:
```

Comment:
In the corresponding prometheus [dashboard](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/aerospike/aerospike-prometheus-overview.json#L48) for this integration, we also have a "aerospike_node_stats_client_connections" metric present that is missing here. Should that be added here?